addons/web_editor/__manifest__.py

```diff
@@ -108,6 +108,10 @@
             'web_editor/static/src/js/wysiwyg/dialog.js',
             'web_editor/static/src/js/frontend/loader.js',
         ],
+        'web.report_assets_common': [
+            'web_editor/static/src/scss/bootstrap_overridden.scss',
+            'web_editor/static/src/scss/web_editor.common.scss',
```

Comment:
if done, we could remove it from `web.assets_frontend` (I think it already includes things in common)